scio-core/src/main/scala/com/spotify/scio/io/FileStorage.scala

```diff
@@ -132,6 +136,22 @@ private class GcsStorage(protected val path: String) extends FileStorage {
   override protected def getObjectInputStream(path: Path): InputStream =
     Channels.newInputStream(gcs.open(GcsPath.fromUri(path.toUri)))
 
+  override protected def getAvroSeekableInput(path: Path): SeekableInput =
+    new SeekableInput {
```

Comment:
Not sure if we should create a separate class for this. For now anonymous class because:
 * no logic - pure delegation
 * no other place where this is useful (so far)